### PR TITLE
Add specs for controller requests and helpers

### PR DESCRIPTION
**Pull request by otherdcn**
As requested, I added controller request and helper specs.

What do you think?

### CODE REVIEW

File: spec/helpers/application_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context "when checking status" do
+    def check_api_football_status
+      api_key = Rails.application.credentials.api_sports.football[:key]
+      api_football_status_url = "https://v3.football.api-sports.io/status"
+
+      ApplicationHelper::APIFootball.get(api_football_status_url, headers: { "x-apisports-key" => api_key })
```
Comment (javiros) [inline spec/helpers/application_helper_spec.rb]: we should probably use a different approach here
1. use VCR cassettes to record the API response so we don't need to make external calls every time we run the specs, it is expensive to do so
2. We should mock the API response and stub the call to the API

we can have a call to work on this if you want

File: spec/helpers/countries_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe CountriesHelper, type: :helper do
+  describe "#countries" do
+    def get_countries(country_code: nil)
```
Comment (javiros) [inline spec/helpers/countries_helper_spec.rb]: same here, we should not make an actual call to the service but mock its response instead

File: spec/helpers/leagues_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe LeaguesHelper, type: :helper do
+  describe "#countries" do
+    def get_leagues(country_code: nil)
```
Comment (javiros) [inline spec/helpers/leagues_helper_spec.rb]: not sure defining methods in specs is a sound approach. we should review this on a 1:1 video call

**Review by javiros — CHANGES_REQUESTED** [review]
Lets work together on these specs, a change of approach is required 👍🏼

File: spec/helpers/leagues_helper_spec.rb (hunk 3 above)
Comment (otherdcn) [reply spec/helpers/leagues_helper_spec.rb]: Yes, let's do a video call. It would really help me.

Whenever you're free.

**Pull request closed without merging**